deploy/50_cloud-ingress-operator.Deployment.yaml

```diff
@@ -54,4 +54,4 @@ spec:
               scheme: HTTP
               port: 8000
             initialDelaySeconds: 25
-            periodSeconds: 15
+            periodSeconds: 60
```

Comment:
What is this used for?

Reply:
This is the time period between each health check. At 15 sec, the pod will crash before the controller can recreate the Load balancer